pkg/pulumiyaml/run_invoke_test.go

```diff
@@ -216,7 +216,7 @@ runtime: yaml
 	requireNoErrors(t, tmpl, diags)
 }
 
-func TestInvokeReturningSecrets(t *testing.T) {
+func TestInvokeReturningSecretsWithReturn(t *testing.T) {
```

Comment:
Is this the best name change? I'm guessing one is testing plain, the other outputs? Or is it something different? It's not super clear to me.

Reply:
Yeah it's not super clear. This test has a `return` property on the invoke on like 229, so the test covers the `if t.Return.GetValue() == "" {` branch above, whereas the other test returns the whole response, covering the rest of the changes in that file.